react-app/src/components/AppSideBar/AppSideBarAPI.ts

```diff
@@ -110,3 +119,30 @@ export function useChainHealthQuery(): {
     requestState,
   };
 }
+
+export function useAppSideBarCommunityStatusQuery(): RequestState<CommunityStatus> {
+  const requestState = useGraphQLQuery<
+    AppSideBarQueryCommunityStatusQuery,
+    AppSideBarQueryCommunityStatusQueryVariables
+  >(AppSideBarQueryCommunityStatus, {
+    // Half average block time
+    pollInterval: 3000,
```

Comment:
3 secs seems too frequent for community status live update

@mitchellchou any live update interval suggestion? 🙏

Reply:
@hochiw 

Reference:
1. Likecoin Average block time: 6.04s
2. The current https://stake.like.co/ seems to listen to the new block event

But I actually think update once per request(e.g. refresh page) should be fine, as those values won't change frequently.